man.Rmd

```diff
@@ -235,16 +316,21 @@ Here's an example from `stringr::str_like()`, which mimics the `LIKE` operator f
 #' * The match is case insensitive by default.
 ```
 
-Finally, it's often particularly hard to write a good description if you've just written the function because the purpose often seems very obvious.
-Do your best, and then come back in a couple of months when you've forgotten exactly what the function does, and re-write the description to jog your memory.
+Basically, if you're going to include an empty line in your description, you'll need to use an explicit `@description` tag.
+
+Finally, it's often particularly hard to write a good description if you've just written the function, because the purpose often seems very obvious.
+Do your best, and then come back later, when you've forgotten exactly what the function does.
+Once you've re-derived what the function does, you'll be able to write a better description.
 
 ### Details
 
-The "details" are just any additional details or explanation that you think your function needs.
+The `@details` are just any additional details or explanation that you think your function needs.
 Most functions don't need details, but some functions need a lot.
-If you have a lot of information to convey, I recommend using markdown headings to break the documentation up into sections.
-Here's an example from `dplyr::mutate()`.
-We've elided some of the details to keep this example short, but you should still get a sense of how we used headings to break up the content in to skimmable chunks:
+If you have a lot of information to convey, it's a good idea to use informative markdown headings to break the details up into manageable sections[^man-11].
+Here's an example from `dplyr::mutate()`. We've elided some of the details to keep this example short, but you should still get a sense of how we used headings to break up the content in to skimmable chunks:
+
+[^man-11]: In older code, you might see the use of `@section title:` which was used to create sections before roxygen2 had full markdown support.
```

Comment:
Moved this to a footnote, which feels more parallel with how we discuss other historical stuff, like `testthat::context()` and `@return`.